iceoryx_hoofs/include/iceoryx_hoofs/cxx/attributes.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+// Copyright (c) 2021-2023 by Apex.AI Inc. All rights reserved.
```

Comment:
Nitpick, but I'm gonna merge anyway ;)

```suggestion
// Copyright (c) 2021 - 2023 by Apex.AI Inc. All rights reserved.
```

Reply:
@mossmaurice there are a plenthora of open PRs where this could be done :)